scripts/create_AR_DeployInst.py

```diff
@@ -417,46 +466,35 @@ def determine_deployment_schedule(
                         e == previous_time]
                 for item in previous_index:
                     if deploy_schedule['DeployInst']['prototypes']['val'][item] == reactor:
-                        num_rxs = math.ceil(
-                            value / reactor_prototypes[reactor][0])
-                        power_gap, value = update_power_demand(power_gap,
-                                                               index,
-                                                               value,
-                                                               num_rxs,
-                                                               reactor_prototypes,
-                                                               reactor)
-                        deploy_schedule = update_di(deploy_schedule,
+                        new_reactors = redeploy_reactors(value,
                                                     reactor,
-                                                    num_rxs,
-                                                    index,
-                                                    reactor_prototypes[reactor][1])
+                                                    reactor_prototypes,
+                                                    deploy_schedule,
+                                                    item)
+                        power_gap, value = update_power_demand(
+                                power_gap, index, value, new_reactors, reactor_prototypes, reactor)
+                        num_rxs[reactor] += new_reactors
         # Deploy new reactors
         if shares is not None:
             for reactor in shares:
-                num_rxs = deploy_with_share(
+                new_reactors  = deploy_with_share(
```

Comment:
See previous comments. `if shares is not None` can be more concise. Either replace `new_reactors` with `new_rxs` or the other way around. (I think `reactors` is better...)